### ISSUE TITLE
Add CONTRIBUTING.md file

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-docs#55]
Add a `CONTRIBUTING.md` file in the root of the repository that contains this content:

```md
See the [Legesher contributing guide](https://github.com/legesher/legesher/blob/master/CONTRIBUTING.md)
```

Simple enough 👍 Please ask to be assigned to this issue so we don't have duplicates 😄

**Comment by machen2** [comment]
Can I work on this issue?

**Comment by madiedgar** [comment]
Absolutely @machen2! I'll assign you to it!

**Comment by madiedgar** [comment]
Closing this as it was resolved with #56 👏
